sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -522,7 +518,7 @@ static void _send_multicast(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
         }
     }
 
-    _send_multicast_over_iface(iface, pkt, netif);
+    _send_multicast_over_iface(iface, pkt);
```

Comment:
Wouldn't this be wrong for https://github.com/RIOT-OS/RIOT/pull/3972/files#diff-74976d45a42eaa063e3561b5fd36f615R507 ?